build/install-operator-sdk.sh

```diff
@@ -1,26 +1,26 @@
 #!/bin/bash
 
 if [ -z "${OPERATOR_SDK_VERSION}" ]; then
-    echo "OPERATOR_SDK_VERSION should be defined as an env varialbe (usually in makefile)"
+    echo "OPERATOR_SDK_VERSION should be defined as an env variable (usually in makefile)"
     exit 1
 fi
 
 if [ -z "${OPERATOR_SDK}" ]; then
-    echo "OPERATOR_SDK should be defined as an env varialbe (usually in makefile)"
+    echo "OPERATOR_SDK should be defined as an env variable (usually in makefile)"
     exit 1
 fi
 
-mkdir -p $(dirname "${OPERATOR_SDK}")
-if [ -x ${OPERATOR_SDK} ]
+mkdir -p "$(dirname "${OPERATOR_SDK}")"
```

Comment:
nested double-quotes there, does that work ok?

Reply:
Yes, this works. the `$( ... )` shields it off. might not be strictly needed though to have it around `$(..)` not 100% sure. But doesn't harm either